ProjectPi/src/server/Server.java

```diff
@@ -0,0 +1,143 @@
+package server;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.net.DatagramPacket;
+import java.net.DatagramSocket;
+import java.net.InetAddress;
+import java.net.SocketException;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.List;
+
+import helper.Transfer;
+
+public class Server {
+	private static final int maxFileNameLength = 100;
+	private static final double pktLossProb = 0;
+
+	private DatagramSocket socket;
+
+	public Server(int port) throws IOException {
+		socket = new DatagramSocket(port);
+	}
+
+	public static void main(String[] args) {
+		try {
+			Server server = new Server(9999);
```

Comment:
It's not really nice to hardcode these kind of parameters. Either read them from a config file, set them in your (gradle) build file, or set them as a constant at the top of your class/method. (in order of best to worst).